src/main/java/com/ait/lienzo/client/core/types/Point2DArray.java

```diff
@@ -84,7 +84,7 @@ public static final Point2DArray fromNFastDoubleArrayJSO(final NFastDoubleArrayJ
         return points;
     }
 
-    Point2DArray(final Point2DArrayJSO jso)
+    public Point2DArray(final Point2DArrayJSO jso)
```

Comment:
Hey @SprocketNYC , changed modifier to public as we need it from the testing side.... thx